bigbluebutton-tests/playwright/polling/poll.js

```diff
@@ -311,7 +312,7 @@ class Polling extends MultiUsers {
     await waitAndClearDefaultPresentationNotification(this.modPage);
     await util.startPoll(this.modPage);
     await this.userPage.waitAndClick(e.pollAnswerOptionBtn);
-    await uploadSinglePresentation(this.modPage, e.questionSlideFileName);
+    await util.uploadSPresentationForTestingPolls(this.modPage, e.questionSlideFileName);
     await this.modPage.waitAndClick(e.publishPollingLabel);
```

Comment:
before performing this action, we need to ensure the presentation has been uploaded successfully and just then publish the poll:

```suggestion
    await this.modPage.hasElement(e.quickPoll);
    await this.modPage.waitAndClick(e.publishPollingLabel);
```